docs/advanced.rst

```diff
@@ -10,12 +10,15 @@ Rally uses `Jinja2 <http://jinja.pocoo.org/docs/dev/>`_ as a template language s
 
 Elasticsearch utilizes Mustache formatting in a few places, notably in `search templates <https://www.elastic.co/guide/en/elasticsearch/reference/7.4/search-template.html>`_ and `Watcher templates <https://www.elastic.co/guide/en/elasticsearch/reference/7.4/actions-email.html>`_. If you are using Mustache in your Rally tracks you must `escape them properly <https://jinja.palletsprojects.com/en/2.10.x/templates/#escaping>`_. See :ref:`put_pipeline` for an example.
 
+.. _advanced_extensions:
+
 Extensions
 """"""""""
 
 Rally also provides a few extensions to Jinja2:
 
 * ``now``: a global variable that represents the current date and time when the template is evaluated by Rally.
+* ``build_flavor``: a global variable that holds build flavor reported by Elasticsearch cluster targetted by Rally.
 * ``days_ago()``: a `filter <http://jinja.pocoo.org/docs/dev/templates/#filters>`_ that you can use for date calculations.
 
 You can find an example in the ``http_logs`` track::
```

Comment:
This might be a bit confusing, since the example only talks about `now` and `days_ago`. Not sure if we may want to add an example with bulld_flavour as well? and specify that the http_log example is for the other two?..

Reply:
Right, I haven't thought about this example below. I moved `build_flavor` variable description lower in https://github.com/elastic/rally/pull/1750/commits/29351b463d84f4bc51f556e6b2454dbdb8983b05. I don't have an example that would make sense outside of serverless context, so I'm not adding it.